frontend/src/constants/storage.ts

```diff
@@ -55,11 +55,13 @@ export const DEFAULT_LIQUID_MANURE_SYSTEM: LiquidManureStorageSystem = {
   separatedLiquidsUSGallons: 0,
   separatedSolidsTons: 0,
   manureStorages: [DEFAULT_LIQUID_MANURE_STORAGE],
+  uuid: '',
 };
 
 export const DEFAULT_SOLID_MANURE_SYSTEM: SolidManureStorageSystem = {
   name: '',
   manureType: ManureType.Solid,
   manuresInSystem: [],
   manureStorage: DEFAULT_SOLID_MANURE_STORAGE,
+  uuid: '',
```

Comment:
Can you have these omit the uuid like other objects do? I don't like having invalid uuids.